src/uniprotkb/components/protein-data-views/ProteomesView.tsx

```diff
@@ -11,29 +11,44 @@ type Props = {
   data?: Xref[];
   isCompact?: boolean;
 };
-const ProteomesView = ({ data, isCompact = false }: Props) => (
-  <ExpandableList descriptionString="proteomes" displayNumberOfHiddenItems>
-    {data?.map((proteome) => (
-      <InfoList
-        key={`${proteome.id}-${proteome.properties?.Component}`}
-        isCompact={isCompact}
-        infoData={[
-          {
-            title: 'Identifier',
-            content: proteome.id && (
-              <Link to={getEntryPath(Namespace.proteomes, proteome.id)}>
-                {proteome.id}
-              </Link>
-            ),
-          },
-          {
-            title: <span data-article-id="proteome_component">Component</span>,
-            content: proteome.properties?.Component,
-          },
-        ]}
-      />
-    ))}
-  </ExpandableList>
-);
+const ProteomesView = ({ data, isCompact = false }: Props) => {
+  const mergedData: { [key: string]: Array<string | undefined> } = {};
+
+  if (data) {
+    data.forEach((proteome) => {
+      const { id, properties } = proteome;
+      if (id && properties?.Component) {
+        mergedData[id] = mergedData[id]
+          ? [...mergedData[id], properties.Component]
+          : [properties.Component];
```

Comment:
Very minor suggestion to avoid array spreading for each new Component addition:

```suggestion
        if (!mergedData[id]) {
          mergedData[id] = [];
        }
        mergedData[id].push(properties.Component);
```